.changeset/few-scissors-agree.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/adapter-vercel': patch
+---
+
+Revert to cjs mode when building for lambda
```

Comment:
This should be updated.

Reply:
good catch, updated this changeset and added another one